.gitmodules

```diff
@@ -50,3 +50,6 @@
 	path = hart-ip
 	url = https://github.com/cisagov/icsnpp-hart-ip
 	branch = main
+[submodule "omron-fins"]
+	path = omron-fins
+	url = https://github.com/cisagov/icsnpp-omron-fins.git
```

Comment:
I think it's best to explicitly specify the branch to track here, too, by adding `branch = main` like the others have.